qgis4/online/dashboard/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM debian:stable
+MAINTAINER Paul Blottiere <blottiere.paul@gmail.com>
+RUN  export DEBIAN_FRONTEND=noninteractive
+ENV  DEBIAN_FRONTEND noninteractive
+
+RUN apt-get update
+RUN apt-get upgrade -y
+RUN apt-get install -y ruby2.3-dev gcc make g++ nodejs
```

Comment:
I'd recommend combining RUN commands into one RUN command, for smaller images.

Reply:
and also for assuring that a change in an install line triggers an apt update as well so we actually install updated packages